packages-content-model/roosterjs-content-model-dom/lib/domUtils/getFocusedPosition.ts

```diff
@@ -0,0 +1,21 @@
+import type { DOMSelection, Position } from 'roosterjs-content-model-types';
+
+/**
+ * Gets the focused position using the DOM Selection provided.
+ * @param selection DOM Selection to use when extracting the focused position
+ * @returns container and offset of the focused position
+ */
+export function getFocusedPosition(selection: DOMSelection): Position | undefined {
```

Comment:
Where will this function be used? If it is not used very widely, maybe we just need to directly read isRevert when we use, so no need to have this public function so no need to add `Position` interface.

Reply:
Agree, I think I will just remove this file.